package proto;

message CMD {
	// 0x0 -- 0x200 框架用的消息
	enum SvrBase {
		CMD_SVR_HEATBEAT			= 0x10;
		CMD_TRANSFER_HEATBEAT		= 0x11;
		CMD_SOCKET_CLIENT_IN		= 0x1;
		CMD_SOCKET_CLIENT_OUT		= 0x2;
		CMD_REGISTER_SERVER_REQ		= 0x3;
		CMD_REGISTER_SERVER_RSP		= 0x4;
		CMD_REGISETR_TRANSFER_REQ   = 0x5;
		CMD_REGISETR_TRANSFER_RSP   = 0x6;
	}
}
